docs/Technical-Documentation/cloud-foundry-db-upgrade.md

```diff
@@ -0,0 +1,120 @@
+# Cloud Foundry, Cloud.gov AWS RDS Database Upgrade
+
+## Process
+
+If you are performing this process for the staging or production, you need to ensure you are performing the changes through the [HHS](https://github.com/HHS/TANF-app) repo and not the [Raft](https://github.com/raft-tech/TANF-app) repo.
+<br/>
+
+### 1. SSH into a backend app in your desired environment
+```bash
+cf ssh tdp-backend-<ENV>
+```
+<br/>
+
+### 2. Create a backup of all the databases in the ENV's RDS instance
+Note: you can get the required field values from `VCAP_SERVICES`.
+```bash
+/home/vcap/deps/0/apt/usr/lib/postgresql/<CURRENT VERSION>/bin/pg_dump -h <HOST> -p <PORT> -d <DB_NAME> -U <DB_USER> -F c --no-acl --no-owner -f <FILE_NAME>.pg
```

Comment:
Can't we use backup script?

Reply:
We cannot. The backup script does not work in it's current state. It does not use the correct DB name which causes the backups to not have the correct (or any) data. Until that bug is fixed, these instructions must be followed.